src/transaction.ts

```diff
@@ -59,12 +65,10 @@ export function deployContract(code: Uint8Array): Action {
  *  or `Uint8Array` instance which represents bytes passed as is.
  * @param gas max amount of gas that method call can use
  * @param deposit amount of NEAR (in yoctoNEAR) to send together with the call
+ * @param stringify Convert input arguments into bytes array.
  */
-export function functionCall(methodName: string, args: Uint8Array | object, gas: BN, deposit: BN): Action {
-    const anyArgs = args as any;
-    const isUint8Array = anyArgs.byteLength !== undefined && anyArgs.byteLength === anyArgs.length;
-    const serializedArgs = isUint8Array ? args : Buffer.from(JSON.stringify(args));
-    return new Action({functionCall: new FunctionCall({methodName, args: serializedArgs, gas, deposit }) });
+export function functionCall(methodName: string, args: Uint8Array | object, gas: BN, deposit: BN, {stringify = stringifyJsonOrBytes} = {}): Action {
+    return new Action({functionCall: new FunctionCall({methodName, args: stringify(args), gas, deposit }) });
 }
```

Comment:
There's actually a destructured new object argument aded here that has a single property called `stringify` on that object, which makes the JSDoc here inaccurate.  For the JSDoc to be accurate, function signature should be:
```suggestion
export function functionCall(methodName: string, args: Uint8Array | object, gas: BN, deposit: BN, stringify = stringifyJsonOrBytes): Action {
    return new Action({functionCall: new FunctionCall({methodName, args: stringify(args), gas, deposit }) });
}
```
Or we could document `opts` in the JSDoc

Reply:
Applied suggestion instead (use argument stringify)